vue-frontend/src/components/home-new/BlogSection.vue

```diff
@@ -0,0 +1,222 @@
+              >
+                {{ blog.pubDate[0] }}
+              </div>
+            </div>
+            <div class="tw-flex tw-flex-col">
+              <div
+                class="tw-mt-[0.9375rem] md:tw-mt-[1.813rem] tw-w-[16.5rem] sm:tw-w-[19.5rem] lg:tw-w-[27.5rem] 2xl:tw-w-[43.875rem] tw-font-inter-semibold tw-text-[1.125rem] md:tw-text-[1.688rem] 2xl:tw-text-[2.125rem] tw-leading-[1.575rem] md:tw-leading-[2.313rem] 2xl:tw-leading-[3.188rem]"
+              >
+                <span
+                  :class="
+                    activeIndex == index
+                      ? 'md:tw-bg-gradient-to-b md:tw-from-[#ff9472] md:tw-to-[#f2709c] md:tw-bg-[length:100%_5px] md:tw-bg-no-repeat md:tw-bg-bottom'
+                      : ''
+                  "
+                  class="tw-pb-[0.3rem] md:hover:tw-bg-gradient-to-b hover:tw-from-[#ff9472] hover:tw-to-[#f2709c] hover:tw-bg-[length:100%_5px] hover:tw-bg-no-repeat hover:tw-bg-bottom"
+                >
+                  {{ blog.title }}</span
+                >
+              </div>
+              <div
+                class="tw-flex tw-flex-row tw-gap-[9rem] lg:tw-gap-72 2xl:tw-gap-[31rem]"
+              >
+                <div
+                  class="tw-mt-[0.9375rem] tw-w-[4.5rem] 2xl:tw-w-[5rem] tw-font-inter-medium tw-text-[0.625rem] md:tw-text-[0.875rem] 2xl:tw-text-[1.125rem] tw-leading-[0.8125rem] md:tw-leading-[1.125rem] 2xl:tw-leading-[1.463rem]"
+                >
+                  {{ blog.author }}
+                </div>
+                <div
+                  class="tw-mt-[0.9375rem] tw-w-[3.25rem] md:tw-w-[4.5rem] 2xl:tw-w-[5.938rem] tw-font-inter-medium tw-text-[0.625rem] md:tw-text-[0.875rem] 2xl:tw-text-[1.125rem] tw-leading-[0.8125rem] md:tw-leading-[1.125rem] 2xl:tw-leading-[1.463rem]"
+                >
+                  Read more
+                </div>
+              </div>
+            </div>
+          </div>
+        </a>
+        <hr
+          v-if="
+            (blogs.length == 2 && index == 1) ||
+            (blogs.length == 3 && index == 2)
+          "
+          class="tw-hidden md:tw-block md:tw-ml-[4.375rem] tw-mt-[1.125rem] 2xl:tw-ml-[5.313rem] md:tw-w-[18.438rem] lg:tw-w-[27.5rem] 2xl:tw-w-[42.375rem] tw-h-[0.063rem] tw-bg-[#C6C6C6]"
+        />
+        <hr
+          v-else
+          class="tw-w-[17.5rem] md:tw-w-[18.438rem] lg:tw-w-[27.5rem] 2xl:tw-w-[42.375rem] tw-h-[0.063rem] tw-ml-[3.20rem] md:tw-ml-[4.375rem] 2xl:tw-ml-[5.313rem] tw-mt-[1.125rem] tw-bg-[#C6C6C6]"
+        />
+
+        <div
+          v-if="activeIndex === index"
+          class="tw-block md:!tw-hidden tw-ml-[3.125rem] tw-mt-[0.5rem] tw-mb-[0.5rem] tw-max-w-[17.625rem] tw-max-h-[10rem] tw-animate-fadeIn"
+        >
+          <a :href="blog.link" target="_blank" v-if="!isClicked">
```

Comment:
I think no need of `isClicked`. You can do it using `activeIndex` and `mobile width`